astraguard/swarm/compressor.py

```
@@ -0,0 +1,358 @@
+
+        # Update state for next delta
+        self.prev_anomaly_sig = anomaly_sig
+
+        return HealthSummary(
+            anomaly_signature=anomaly_sig,
+            risk_score=risk_score,
+            recurrence_score=recurrence_score,
+            timestamp=timestamp,
+        )
+
+        return summary
```

Ruff (0.14.10)

Undefined name `summary`

(F821)